src/adr/0018-jmap-new-specs.md

```diff
@@ -25,13 +25,14 @@ We decided to do as follow:
 * Implement the new JMAP request structure with the [echo](https://jmap.io/spec-core.html#the-coreecho-method) method
 * Implement authentication and session of the new JMAP protocol
 * Implement protocol-level error handling
-* Port existing mailbox methods of `jmap-draft` to `jmap`
-* Port existing email methods of `jmap-draft` to `jmap`
-* Port existing vacation methods of `jmap-draft` to `jmap`
+* Duplicate and adapt existing mailbox methods of `jmap-draft` to `jmap`
+* Duplicate and adapt existing email methods of `jmap-draft` to `jmap`
+* Duplicate and adapt existing vacation methods of `jmap-draft` to `jmap`
 * Support uploads/downloads
 
-Then when we finish to port our existing methods to the new JMAP specifications, we can implement the new features that we are missing and are absolutely needed:
+Then when we finish to port our existing methods to the new JMAP specifications, we can implement these new features:
 
+* Identities
```

Comment:
Identities is stored by OpenPaaS, we have no benefits of indentities in James.

Maybe you meant 'account'?

Reply:
It was related to @rouazana comment, and it is in the specs? https://jmap.io/spec-mail.html#identities

Or did I misunderstand something?

Reply:
`BTW in both cases you miss account/identities`

You grepped the worng part of the sentence, that's all ^^

https://jmap.io/spec-mail.html#identities is implmented directly by openPaaS, so Linagora side there is little need to implement it in JMAP as far as I know.

Reply:
I really meant accounts + identities. I know that we implemented identities in OpenPaaS, but for me it's a temporary workaround, I don't see why we should derogate to the spec here.

Reply:
And actually in a more general way, other implementations should be able to use those. I think the point is fair